bodhi/tests/messages/schemas/test_update.py

```diff
@@ -15,7 +15,7 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 """Unit tests for the update message schemas."""
 
-import unittest
+
```

Comment:
I don't know why I didn't spot this earlier: this additional empty line makes flake8 complain (which is run during our automatic tests) -- there need to be two blank lines between the docstring of a module and the imports, this makes it three. Would you please remove it? Thanks!